pycsw/ogc/api/records.py

```diff
@@ -1035,8 +1040,25 @@ def record2json(record, url, collection, stac_item=False):
                 [minx, miny]
             ]]
         }
+        record_dict['extent'] = {
```

Comment:
As it happens, `extent` is no longer a part of the OARec core record model.  Can we remove?

Reply:
ah i was wondering about this, so properties.extent.temporal has moved to feature.time, and properties.extent.geospatial to feature.geometry?

Reply:
yes.  `feature.time` and `feature.geometry` are single spatiotemporal definitions.  Any additional spatiotemporal definitions are left to downstream profiles.